### ISSUE TITLE
which method provide Youtube Data Api v3 key and we see black screen with audio only. not any buttons.

### ISSUE THREAD

**Issue by GauravMeghanathiWeDoApps** [PierfrancescoSoffritti/android-youtube-player#1039]
# Bug Report
 we are not able to play youtube video just we are hearing the audio only and see the blackscreen not even any controls of youtube player. as well as there is i review your code but We could not see where to set Youtube Data Api key from google server or not mention in document. kindly please update method as well as screen shot of where to set. 
### Environment details

Android Version: 
API Version: 30 to 34 

Tested devices:

Youtube Player Library Version: 

### Steps to reproduce the bug
---
name: Bug report
about: Use this template for reporting bugs
labels: bug
---

 

# Bug Report

### Environment details

Android Version: 
API Version:  30 t0 34

Tested devices:

Youtube Player Library Version: 

### Additional information
<!-- Provide here further information that might help understand the issue or reproduce the bug, e.g. 
- standalone code snippets of implementation that can copy-pasted and work without any missing classes
- screenshot(s) of visualising the bug
- error logs
- any useful background infromation

**Comment by PierfrancescoSoffritti** [comment]
The YouTube Data API is not required for this library.

Are you seeing the black screen on an emulator or real device?

**Comment by Naguchennai** [comment]
i also same issue .. you added this?

    @Override
    public void onDestroy() {
        super.onDestroy();
        youTubePlayerView.release();
    }

**Comment by GauravMeghanathiWeDoApps** [comment]
@Override
public void onDestroy() {
    super.onDestroy();
    youTubePlayerView.release();
}

yes i added this and i am also see the black screen  i have test on real device real me pro 6 5g as well as emulator both getting same result of output.

**Comment by PierfrancescoSoffritti** [comment]
Does the black screen happens when you enter fullscreen?

**Comment by Naguchennai** [comment]
1. if i go to youtube player activity its working fine
2. come back and go to any other activity and go back to youtube player activity its black screen..

**Comment by PierfrancescoSoffritti** [comment]
mmm are you able to replicate this in the sample app? if not, the problem is probably in your code.

**Comment by Naguchennai** [comment]
> mmm are you able to replicate this in the sample app? if not, the problem is probably in your code.

Thank you finally i found the issue .. Problem is i also using another webview library .. two webview library causing this issue

**Comment by PierfrancescoSoffritti** [comment]
Thanks for sharing! can I ask what is the other webview library you're using?

**Comment by Naguchennai** [comment]
implementation 'com.github.delight-im:Android-AdvancedWebView:v3.0.0'

after i didsbale mWebView.onResume(); .. its working fine no more black screen 

       @SuppressLint("NewApi")
    @Override
    protected void onResume() {
        super.onResume();
       // mWebView.onResume();
    }

**Comment by GauravMeghanathiWeDoApps** [comment]
yes  I am just using your library  implementation 'com.pierfrancescosoffritti.androidyoutubeplayer:core:12.1.0'
